scripts/rebuild-deb.sh

```diff
@@ -48,9 +52,13 @@ esac
 case "${DUNE_PROFILE}" in
   devnet)
     MINA_DEB_NAME="mina-berkeley"
-    ;;
+    DEB_SUFFIX=""
+   ;;
   *)
-    MINA_DEB_NAME="mina-berkeley-${DUNE_PROFILE}"
+    # use dune profile as suffix but replace underscode to dashes sor deb builder won't complain
```

Comment:
```suggestion
    # use dune profile as suffix but replace underscore to dashes so deb builder won't complain
```

Reply:
Also I suggest to capture `${DUNE_PROFILE//_/-}` into a new variable and reuse below.